### ISSUE TITLE
Chat Service implementation

### ISSUE THREAD

**Issue by simsulison** [WinterSpringProject/PROJECT-GLFP#33]
Spring에서 Websocket과 STOMP protocol을 이용하여 채팅 서비스를 구현하는 것을 목표로 한다.

**Comment by gyumnnii** [comment]
질문 있습니다!

stomp를 사용하신다면 publish/subscribe 구조로 소켓 구현하시는 것이 맞나요?

해당 프로토콜을 사용하면 프론트엔드도 라이브러리를 맞춰 사용하는 것으로 이전에 협의가 되어있어야 할텐데 해당 협의도 이루어졌나요?

**Comment by simsulison** [comment]
네 pub/sub 구조로 구현하는 것이고, 백엔드 readme 참고하시면 기술스택에 stomp프로토콜을 사용하기로 되어있긴 합니다. 

해당 프로토콜을 공부하다보니 오히려 프론트쪽에서 신경 쓸 부분이 많다는 것을 확인하여 어제 회의를 통해 전달한 상태입니다!

**Comment by bluewink** [comment]
누가 채팅요청을 보냈는지 구분하기 위해서 session을 이용합니다.

## references
- https://www.devglan.com/spring-boot/spring-session-stomp-websocket

**Comment by simsulison** [comment]
전체 채팅방(실시간 채팅방) 구현 먼저 진행해보겠습니다.
